app/components/login-form.js

```diff
@@ -4,55 +4,49 @@ import { tracked } from '@glimmer/tracking';
 import { inject as service } from '@ember/service';
 
 export default class LoginFormComponent extends Component {
-    @service router;
-    @tracked emailError;
-    @tracked passwordError;
-    @tracked email;
-    @tracked password;
-    @tracked errorMessage
+  @service router;
+  @tracked emailError;
+  @tracked passwordError;
+  @tracked email;
+  @tracked password;
+  @tracked errorMessage;
 
-    validateEmail(email) {
```

Comment:
Why this much changes in this file?

Reply:
by mistakes i will correct it hereafters